cmd/habitat/node/fs/handlers.go

```diff
@@ -0,0 +1,100 @@
+package fs
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/eagraf/habitat/cmd/habitat/api"
+	"github.com/eagraf/habitat/pkg/ipfs"
+	"github.com/eagraf/habitat/structs/ctl"
+)
+
+// CAFS stands for Content-Addressable File System. This interface is an abstraction
+// of file systems like IPFS
+type CAFS interface {
```

Comment:
to integrate Sashank's stuff, we'll have to implement this https://pkg.go.dev/github.com/ipfs/go-datastore?utm_source=godoc#Datastore over CAFS right?

Reply:
yeah I think so